sentry/src/test/java/io/sentry/SentryThreadFactoryTest.kt

```diff
@@ -25,34 +21,37 @@ class SentryThreadFactoryTest {
   @Test
   fun `when getCurrentThreads is called, not empty result`() {
     val sut = fixture.getSut()
-    val threads = sut.getCurrentThreads(null)
```

Comment:
`l` some tests here used to implicitly have `attachStacktrace=true`, now we're explicitly passing `false`. Does that matter?